srcpkgs/gitea/template

```diff
@@ -42,31 +42,12 @@ make_dirs="/var/lib/gitea 0755 _gitea _gitea
 conf_files="/etc/gitea.conf"
 
 pre_build() {
-	make generate
-}
-
-pre_install() {
-	sed -i \
-		'/RUN_USER/s:git:_gitea:
-		s:666:660:
-		/RUN_MODE/s:dev:prod:
-		/DB_TYPE/s:mysql:sqlite3:
-		s:ROOT =:& /var/lib/gitea/repositories:
-		s:tmp/local-repo:/var/lib/gitea/&:
-		s:tmp/local-wiki:/var/lib/gitea/&:
-		/TEMP_PATH/s:data/tmp/uploads:/var/lib/gitea/tmp/uploads:
-		/LFS_CONTENT/s:data/lfs:/var/lib/gitea/lfs:
-		s:data/gitea.db:/var/lib/gitea/&:
-		s:indexers/issues.bleve:/var/lib/gitea/&:
-		s:data/avatars:/var/lib/gitea/&:
-		s:SSH_ROOT_PATH =:& /var/lib/gitea:
-		s:STATIC_ROOT_PATH =:& /var/lib/gitea:
-		s:^ROOT_PATH =:& /var/log/gitea:' custom/conf/app.ini.sample
+	go mod vendor
+	CGO_CFLAGS= CGO_LDFLAGS= make generate
```

Comment:
if you're using the provided tarballs, this step should no longer be needed.

Reply:
You are correct, checked.